### PR TITLE
Fix: stderr response

### PR DESCRIPTION
**Pull request by Meldiron**
With successful execution, stderr was ignored. Now both stdout and stderr are ALWAYS present, as intended.

+ I noticed small bug when empty object is translated to empty array. Fixed that.

I also improved maintainability of tests a little to prevent code duplication.
With that I added tests for some more edge cases: stderr, json object response.

### CODE REVIEW

File: app/http.php [hunk 1]
```diff
@@ -728,29 +728,21 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
...
-                default:
-                    $stderr = $executorResponse['stderr'] ?? '';
-                    $stdout = $executorResponse['stdout'] ?? '';
-                    break;
+            if ($statusCode >= 200 && $statusCode < 300) {
+                $res = $executorResponse->response ?? '';
+                if (is_array($res)) {
+                    $res = json_encode($res, JSON_UNESCAPED_UNICODE);
+                }
+                if (is_object($res)) {
```
Comment (christyjacob4) [inline app/http.php:743]: Better to use `else if` here

File: app/http.php [hunk 2]
```diff
@@ -728,29 +728,21 @@ function (string $runtimeId, string $payload, array $variables, int $timeout, st
...
-                    $res = $executorResponse['response'] ?? '';
-                    if (is_array($res)) {
-                        $res = json_encode($res, JSON_UNESCAPED_UNICODE);
-                    }
-                    break;
-                default:
-                    $stderr = $executorResponse['stderr'] ?? '';
-                    $stdout = $executorResponse['stdout'] ?? '';
-                    break;
+            if ($statusCode >= 200 && $statusCode < 300) {
```
Comment (christyjacob4) [inline app/http.php:738]:
```suggestion
            if ($statusCode >= 200 && $statusCode < 500) {
```
Why 300? Anything < 500 is a valid response right ?

File: app/http.php (hunk 1 above)
Comment (Meldiron) [reply app/http.php:743]: Added

**Review by christyjacob4 — APPROVED** [review]
